CMakeLists.txt

```diff
@@ -30,6 +30,9 @@ option(USE_PNG "Use PNG?" off)
 option(USE_JASPER "Use Jasper?" off)
 option(USE_OPENJPEG "Use OpenJPEG?" off)
 option(USE_AEC "Use AEC?" off)
+option(FTP_TEST_FILES "Fetch and test with files on FTP site." ON)
+option(FTP_LARGE_TEST_FILES "Fetch and test with very large files on FTP site." OFF)
+option(FTP_EXTRA_TEST_FILES "Fetch even more large files from FTP and test them." OFF)
```

Comment:
Also add these lines:

```
# Developers can use this option to specify a local directory which
# holds the test files. They will be copied instead of fetching the
# files via FTP.
SET(TEST_FILE_DIR "." CACHE STRING "Check this directory for test files before using FTP.")
message(STATUS "Finding test data files in directory ${TEST_FILE_DIR}.")

```